### PR TITLE
pin rasterio <=1.3.11

### PR DESCRIPTION
**Pull request by TC-FF**
### Temporary fix until  xarray-contrib/xvec#85 is resolved
- [ ] CHANGELOG.rst has been updated (with summary of main changes).

### What kind of change does this PR introduce?
pin rasterio <=1.3.11

### CODE REVIEW

File: environment-dev.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ dependencies:
   - pooch >=1.8.0
   - pydantic >=2.0,<2.5.3  # FIXME: Remove pin once our dependencies (xclim, xscen) support pydantic 2.5.3
   - pyyaml
-  - rasterio
+  - rasterio <=1.3.11
```
Comment (Zeitsperre) [inline environment-dev.yml:21]:
```suggestion
  - rasterio <=1.3.11  # temporary pin until https://github.com/xarray-contrib/xvec/issues/85 is addressed.
```

File: environment.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ dependencies:
   - pystac
   - pystac-client
   - pyyaml
-  - rasterio
+  - rasterio <=1.3.11
```
Comment (Zeitsperre) [inline environment.yml:22]:
```suggestion
  - rasterio <=1.3.11  # temporary pin until https://github.com/xarray-contrib/xvec/issues/85 is addressed.
```

File: pyproject.toml [hunk 3]
```diff
@@ -49,7 +49,7 @@ dependencies = [
   "pystac",
   "pystac-client",
   "pyyaml",
-  "rasterio",
+  "rasterio <=1.3.11",
```
Comment (Zeitsperre) [inline pyproject.toml:52]:
```suggestion
  "rasterio <=1.3.11",  # temporary pin until https://github.com/xarray-contrib/xvec/issues/85 is addressed.
```

**Review by Zeitsperre — APPROVED** [review]
LGTM

**Comment by martinfleis** [comment]
Hey, I see that you refer to the xvec issue here. If you unpin rasterio, do you still encounter any issues? I closed that one as I am unable to reproduce it anymore but I may just lack a proper reproducible example.

**Pull request merged**